code/game/objects/explosion.dm

```diff
@@ -40,28 +40,22 @@
 
 		if(!silent)
 			var/frequency = get_rand_frequency()
-			for(var/mob/M in player_list)
+			var/sound/explosion_sound = sound(get_sfx("explosion"))
+			var/sound/global_boom = sound('sound/effects/explosionfar.ogg')
+
+			for(var/P in player_list)
+				var/mob/M = P
 				// Double check for client
 				if(M && M.client)
 					var/turf/M_turf = get_turf(M)
 					if(M_turf && M_turf.z == epicenter.z)
 						var/dist = get_dist(M_turf, epicenter)
 						// If inside the blast radius + world.view - 2
 						if(dist <= round(max_range + world.view - 2, 1))
-							M.playsound_local(epicenter, get_sfx("explosion"), 100, 1, frequency, falloff = 5) // get_sfx() is so that everyone gets the same sound
+							M.playsound_local(epicenter, null, 100, 1, frequency, falloff = 5, S = explosion_sound)
 						// You hear a far explosion if you're outside the blast radius. Small bombs shouldn't be heard all over the station.
-						else if(dist <= far_dist)

```

Comment:
This looks like it changes a lot, but it doesn't; basically, if you were close to the explosion, it'd play the loud "boom"; if you were outside of the major blast radius, then you'd hear the muffled "boom".

That said, then, if you were slightly outside of this range, you'd hear a global soft "boom".

This seemed redundant; if everyone outside of the big blast radius is meant to hear a muffled boom, then why loop through the player list twice?